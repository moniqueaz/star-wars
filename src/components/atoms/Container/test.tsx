import { render } from '@testing-library/react';

import { Container } from './';

describe('<Container />', () => {
  it('should render correctly', () => {
    const { container } = render(<Container>Conteudo</Container>);

    expect(container).toBeInTheDocument();

    expect(container.firstChild).toMatchSnapshot();
  } );
} );
