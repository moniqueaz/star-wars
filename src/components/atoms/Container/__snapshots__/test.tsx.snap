// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> should render correctly 1`] = `
.c0 {
  padding: 10px;
  margin: 0 auto;
  max-width: 1250px;
  width: 100%;
  background-color: #EAEAEA;
}

<div
  class="c0"
>
  Conteudo
</div>
`;
